src/android/app/src/main/res/layout/game_about_dialog.xml

```diff
@@ -0,0 +1,131 @@
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/constraintLayout"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHeight_min="140dp"
+            app:layout_constraintStart_toEndOf="@id/game_icon"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <TextView
+                android:id="@+id/game_title"
+                style="?attr/textAppearanceTitleMedium"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textAlignment="viewStart"
+                android:textSize="20sp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:text="Game Title" />
+
+            <TextView
+                android:id="@+id/game_author"
+                style="?attr/textAppearanceBodyMedium"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:layout_constraintStart_toStartOf="@id/game_title"
+                app:layout_constraintTop_toBottomOf="@id/game_title"
+                tools:text="Game Author" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <LinearLayout
+            android:id="@+id/horizontal_layout"
+            style="@style/ThemeOverlay.Material3.Button.IconButton.Filled.Tonal"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:gravity="start|center"
+            android:layout_marginTop="16dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/game_icon">
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/game_play"
+                style="@style/Widget.Material3.Button.Icon"
+                android:layout_width="167dp"
+                android:layout_height="wrap_content"
+                android:contentDescription="@string/play"
+                android:focusedByDefault="true"
+                android:text="@string/play"
+                app:icon="@drawable/ic_play"/>
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/game_settings"
+                style="@style/Widget.Material3.Button.IconButton.Filled.Tonal"
+                android:layout_width="48dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:textColor="?attr/colorAccent"
+                app:icon="@drawable/ic_settings"
+                app:iconGravity="textStart" />
```

Comment:
The placement of this button seems to indicate that it should open game-specific settings, however it instead opens the regular shared settings menu.

Reply:
I think the author was working on per-game-settings so he just added it and didn't bother to remove it after making the original PR

Personally, I know its confusing but  I like the quick access to settings and maybe some other person can add per-game settings on the future?

Anyway, if you wish I could disable it for now